opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreAncMemberMapActivity.java

```diff
@@ -152,8 +154,11 @@ private void featureClicked(@NonNull Feature feature) {
         String basicServiceProvided = feature.getStringProperty("Basic Service Provided ANC_BEmONC_PPC_QI_MIP_ENC_FP_HIV_RMNCH_integration");
         String cEmONC = feature.getStringProperty("CEmONC");
         String ownership = feature.getStringProperty("Ownership");
+
         if (StringUtils.isNotBlank(facilityName) && StringUtils.isNotBlank(status)) {
-            BaseAncRespondersCallDialogFragment.launchDialog(this, null, null, false,
+            model.setResponderName(null);
+            model.setResponderPhoneNumber(null);
```

Comment:
Quick question, why set these as null? Is there a place that they are used?

Reply:
Yeah, So the idea is to launch the call dialog without those values... 
This I lifted from the existing implementation. See on the changed code on the left hand side

Reply:
Okay.